config/locales/client.en.yml

```diff
@@ -61,8 +61,4 @@ en:
       explain_label: "Include query plan?"
       save_params: "Set Defaults"
       reset_params: "Reset"
```

Comment:
@techAPJ These vars are unused so is it safe to delete them from the source file?

Reply:
Yes, unused vars can be deleted safely.